sunpy/map/tests/test_mapbase.py

```diff
@@ -495,40 +500,44 @@ def test_shift_applied(generic_map):
                                 generic_map.reference_coordinate.Ty)
     x_shift = 5 * u.arcsec
     y_shift = 13 * u.arcsec
-    shifted_map = generic_map.shift(x_shift, y_shift)
+    shifted_map = generic_map.shift_reference_coord(x_shift, y_shift)
```

Comment:
Instead of using the new method, I would keep the tests as `shift()` here (and filter/catch the warnings) so we're still testing both methods until the deprecated one is removed.

Reply:
Ah that's a good point. I'll fix that.